packages/front-end/components/Features/ConditionInput.tsx

```diff
@@ -151,6 +154,7 @@ export default function ConditionInput(props: Props) {
                     { label: "is not in the list", value: "$nin" },
                     { label: "exists", value: "$exists" },
                     { label: "does not exist", value: "$notExists" },
+                    //TODO: Should is in the group be in here, too?
```

Comment:
@jdorn I'm thinking we should include the "is/is not in group" for enums. Any thoughts?

Reply:
I'm fine adding it for enums, although I doubt it will be used very often.

We should only show the `is in the group` option when there are actually saved groups for the selected attribute.

Reply:
10-4. Makes sense. Just updated this so enums are included, and we're only showing the `is in the group` or `is not in the group` options when an organization has groups.